### ISSUE TITLE
scrape after link opens up in new tab

### ISSUE THREAD

**Issue by rahulsingh303** [MontFerret/ferret#640]
I am trying to scrape a webpage where after clicking on a link , it opens up a new page . and then I need to scrape the new page . But seems ferret is unable to update the new context of the page , has anyone been able to scrape for a similar situation ?

you can see this behavior on booking.com where after click on any hotel it opens up a new tab . 

Regards
Rahul

**Comment by ziflex** [comment]
You are right, at this moment there is no such functionality. I will look into this after this upcoming release.

**Comment by polaMikhail** [comment]
@ziflex have you got a chance to tackle this one, I would like to give it a shot.

**Comment by ziflex** [comment]
I will try to come up with something after the holidays.

**Comment by ziflex** [comment]
Created a dedicated issue.
